lib/facility_factory.rb

```diff
@@ -0,0 +1,36 @@
+class FacilityFactory
+  
+  def initialize
+  end
+
```

Comment:
Since you do not need to initialize the `FacilityFactory` objects with any state (attributes), you don't need this `initialize` method. We really only need to have that when we want to instantiate an object with state from the very start.